j2k/newSrc/org/jetbrains/kotlin/j2k/NewCodeBuilder.kt

```diff
@@ -313,6 +324,15 @@ class NewCodeBuilder {
                 else -> {
                 }
             }
+            if (type is JKClassType && type.parameters.isNotEmpty()) {
```

Comment:
I've also added this, so will look at merge time, also duplicates logic from method rendering